modules/store/src/models.ts

```diff
@@ -33,6 +33,9 @@ export interface StoreFeature<T, V extends Action = Action> {
   metaReducers?: MetaReducer<T, V>[];
 }
 
-export interface Selector<T, V> {
-  (state: T): V;
-}
+export type Selector<State, Result> = (state: State) => Result;
+
+export type SelectorWithProperty<State, Property, Result> = (
```

Comment:
Property should be plural because we're passing a set of them. `Props` or `Properties`?

Reply:
I would say `Props`?